tests/wicked/before_test.pm

```diff
@@ -89,11 +90,16 @@ sub run {
             assert_script_run('./autogen.sh ',       timeout => 600);
             assert_script_run('make ; make install', timeout => 600);
         } elsif (my $wicked_repo = get_var('WICKED_REPO')) {
-            zypper_ar($wicked_repo, name => 'wicked_repo', no_gpg_check => 1);
+            record_info('REPO', $wicked_repo);
+            zypper_ar($wicked_repo, params => '-n wicked_repo', no_gpg_check => 1);
             zypper_call('in --force -y --from wicked_repo --allow-vendor-change  --allow-downgrade  wicked', log => 1);
-            validate_script_output('zypper ps  --print "%s"', /^\s*$/);
+            # Restart services
+            script_run('rcwickedd restart');
+            script_run('rcwicked restart');
+            validate_script_output('zypper ps  --print "%s"', qr/^\s*$/);
             if (my $commit_sha = get_var('WICKED_COMMIT_SHA')) {
-                validate_script_output(q(head -n 1 /usr/share/doc/packages/wicked/ChangeLog | awk '{print $2}'), /^$commit_sha$/);
+                record_info('COMMIT', $commit_sha);
```

Comment:
I would switch the order here

Reply:
I mean first verify and only than display record_info

Reply:
ok